rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -180,18 +181,18 @@ class DebruijnInterpreter[M[_], F[_]](
     else {
 
       // Collect errors from all parallel execution paths (pars)
-      terms.zipWithIndex.toVector
-        .parTraverse {
-          case (term, index) =>
-            eval(term)(env, split(index))
-              .map(_ => none[Throwable])
-              .handleError(_.some)
-        }
-        .map(_.flattenOption)
-        .flatMap(aggregateEvaluatorErrors)
+      parTraverseSafe(terms.zipWithIndex.toVector) {
+        case (term, index) =>
+          eval(term)(env, split(index))
+      }
     }
   }
 
+  private def parTraverseSafe[A, B](xs: Vector[A])(op: A => M[Unit]): M[Unit] =
```

Comment:
What is type parameter `B` used for? Is it `Unit` or `M[Unit]`?

Reply:
It's not used, I started with B but replaced it with Unit. Thanks!